lib/issuer.js

```diff
@@ -188,33 +230,206 @@ api.claim = function(actor, id, callback) {
   });
 };
 
-bedrock.events.on('bedrock-mongodb.ready', function(callback) {
-  async.auto({
-    createIndexes: function(callback) {
-      // create indexes for claimed status
-      database.createIndexes([{
-        collection: store.provider.name,
-        fields: {recipient: 1, 'credential.sysState': 1, id: 1},
-        options: {unique: true, background: false}
-      }, {
-        collection: store.provider.name,
-        fields: {issuer: 1, 'credential.sysState': 1, id: 1},
-        options: {unique: true, background: false}
-      }], callback);
+// TODO: more acceptable formats?
+function checkHeaders(req, res, next) {
+  var acceptableContentTypes = [
+    'application/json',
+    'application/ld+json',
+    'text/html'
+  ];
+  if(!req.accepts(acceptableContentTypes)) {
+    res.status(406);
+    res.write(JSON.stringify({
```

Comment:
I think `res.json()` can just take a JS object directly (no need to make the extra stringify call).